app/views/users/show.html.haml

```diff
@@ -0,0 +1,44 @@
+%header.entry-header
+  - if @user.admin?
+    %h1.entry-title.is-admin=  @user.email
+  - else
+    %h1.entry-title=  @user.email
+
+
+.entry-content
+  %p
+    %b #{t('.email')}:
+    = @user.email
+
+  - if @user.last_sign_in_at.blank? && @user.current_sign_in_at.blank?
+    %p  #{@user.email} #{t('.user_has_never_signed_in')}
+
+  - else
+    %p
+      %b #{t('.last_login')}:
+      #{ most_recent_login_time @user}
+      (#{time_ago_in_words(most_recent_login_time @user)} ago)
+    %p
+      %b #{t('.last_login_ip')}:
+      #{ most_recent_login_ip @user}
```

Comment:
Agreed.  

@thesuss  -- thoughts on this?  Has there ever been a situation when it was helpful to see the IP address, or might be?  

I'd be happy to remove this for now and then also make sure we explicitly ask the client about it.   Seems like a pretty small thing and would be easy to add back in.  (would be a good 'easy change' for a novice.)

I'll make a note in the agenda for the client meeting and also put a chore into PivotalTracker so it's addressed at some point.)